sqs.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_sqs_queue" "keycloak_to_alerts_concierge_user_updates" {
+  name = "keycloak-${var.environment}-alerts-concierge-user-updates"
+  sqs_managed_sse_enabled = true
+}
+
+# Allow Keycloak ECS to publish messages to SQS
+resource "aws_sqs_queue_policy" "keycloak_to_alerts_concierge_user_updates" {
+  queue_url = aws_sqs_queue.keycloak_to_alerts_concierge_user_updates.id
+  policy    = data.aws_iam_policy_document.keycloak_to_alerts_concierge_user_updates_policy.json
+}
+data "aws_iam_policy_document" "keycloak_to_alerts_concierge_user_updates_policy" {
+  statement {
+    sid = "AllowSendFromKeycloakECS"
+    principals {
```

Comment:
@ianwestcott Do we need to define the principal in addition to the specific condition below?

Reply:
This is what we're doing in other places, but Terraform's documentation implies that you can get away with a wildcard since the condition is sufficiently restrictive. I like the idea of being as restrictive as possible, so let's leave it as is, and if we have problems with this in testing then we can broaden the principal?